src/lib/webpack/webpack-base-config.js

```diff
@@ -239,7 +239,7 @@ export default function (env) {
 			})
 		] : []),
 
-		devtool: isProd ? 'source-map' : 'cheap-module-eval-source-map',
+		devtool: isWatch ? 'cheap-module-eval-source-map' : 'source-map',
```

Comment:
@lukeed why did you opt for `isWatch` rather than `isProd` here? this decision appears to break my source maps when building with `--no-production`

Reply:
You would have ended up with `cheap-module-eval-source-map` without this change too

Reply:
Sounds like you should just force `devtool: 'source-map'` in your config file – you don't seem to have a typical case so I don't think anything needs to change here.